packages/core/package.json

```diff
@@ -25,7 +25,8 @@
     "fp-ts": "^2.11.5",
     "io-ts": "^2.2.16",
     "io-ts-reporters": "^2.0.1",
-    "io-ts-types": "^0.5.16"
+    "io-ts-types": "^0.5.16",
+    "ora": "^6.1.2"
```

Comment:
This shouldn't be in the Core, it's a node only dependency and caused us issues in the past.
This is purely used in the INIT so i don't see a reason to have it here.

Reply:
It's automatically added on lerna bootstrap I believe, I'll remove it

Reply:
is ora used in the core anywhere?

Reply:
no